MobileCenterPush/MobileCenterPushTests/MobileCenterPushTests.m

```diff
@@ -0,0 +1,139 @@
+#import <Foundation/Foundation.h>
+#import <XCTest/XCTest.h>
+#import <OCMock/OCMock.h>
+#import <OCHamcrestIOS/OCHamcrestIOS.h>
+
+#import "MSService.h"
+#import "MSServiceAbstract.h"
+#import "MSServiceInternal.h"
+
+#import "MSPush.h"
+#import "MSPushPrivate.h"
+#import "MSPushInternal.h"
+#import "MSMockPushDelegate.h"
+#import "MSPushLog.h"
+
+static NSString *const kMSTestAppSecret = @"TestAppSecret";
+static NSString *const kMSTestDeviceToken = @"TestDeviceToken";
+
+@interface MSPushTests : XCTestCase <MSPushDelegate>
```

Comment:
I used tests from Analytics project as example. I'm rewriting tests related to delegates